### PR TITLE
Improve and cleanup core styles, improve seven skin

### PR DESCRIPTION
**Pull request by johnBartos**
- New seven skin
    - Most of it is now part of core styles
- Improve old skins, fix existing bugs

Fixes #
JW7-2534

### CODE REVIEW

File: src/css/imports/playlist.less [hunk 1]
```diff
@@ -1,45 +1,40 @@
 @import "vars";
 
-
 .jw-playlist-container {
+    left: -51%;
```
Comment (pajong) [inline src/css/imports/playlist.less]: What is this -51%?

File: src/css/imports/tooltip.less [hunk 2]
```diff
@@ -92,7 +92,7 @@
         line-height: 1.5em;
         color: @inactive-color;
         padding: 0 .5em;
```
Comment (pajong) [inline src/css/imports/tooltip.less]: Is this ```0 0.5em``` or ```0.5em``` with a typo?

File: src/css/imports/playlist.less (hunk 1 above)
Comment (johnBartos) [inline src/css/imports/playlist.less]: It moves the playlist container over - it's something that's done in every skin, but this value is specifically for the core styling (i.e. seven)

File: src/css/skins/roundster.less [hunk 3]
```diff
@@ -1,4 +1,7 @@
 @import "../imports/mixins";
+//@import "../imports/vars";
```
Comment (pajong) [inline src/css/skins/roundster.less]: Remove unused comments

File: src/css/skins/six.less [hunk 4]
```diff
@@ -114,11 +119,12 @@
         }
 
         ::-webkit-scrollbar {
-            border-radius: .25em;
+            border-radius: .25em !important;
```
Comment (pajong) [inline src/css/skins/six.less]: We try not to use ```!important``` - is there a way to not use important and achieve the same goal?

File: src/css/skins/six.less [hunk 5]
```diff
@@ -246,7 +241,7 @@
         background-size: @def-background-size;
         border: @def-border;
         border-radius: .3em;
-        padding: @ui-padding @ui-corner-round;
+        //padding: @ui-padding @ui-corner-round;
```
Comment (pajong) [inline src/css/skins/six.less]: Remove unused comment

**Pull request merged**